learningresources/models.py

```diff
@@ -19,6 +19,18 @@
 log = logging.getLogger(__name__)
 
 
+def static_asset_basepath(asset, filename):
+    """
+    Creates folder base path for given asset.
+    """
+    return 'assets/{org}/{course_number}/{run}/{filename}'.format(
+        org=asset.course.org,
+        course_number=asset.course.course_number,
```

Comment:
I don't think we currently don't verify that these are free of slashes. Is it possible to do a check and raise an exception if a slash is found? There might be a better place to do this, not sure